crypto/fipsmodule/evp/p_pqdsa.c

```diff
@@ -6,10 +6,9 @@
 #include <openssl/mem.h>
 
 #include "../crypto/evp_extra/internal.h"
-#include "../crypto/fipsmodule/ml_dsa/ml_dsa.h"
+#include "../crypto/ml_dsa/ml_dsa.h"
 #include "../crypto/internal.h"
-#include "../delocate.h"
-#include "../pqdsa/internal.h"
+#include "../../pqdsa/internal.h"
 
 // PQDSA PKEY functions
```

Comment:
should these stay in the FIPS module or be moved to evp_extra or similar.

not sure if we should be including non-fipsmodule stuff in the FIPS module.

Reply:
yup, also just noticed this! [e2f24df](https://github.com/aws/aws-lc/pull/2104/commits/e2f24df3675c29ae3585284378accc16add04e7d)